docs/configuration_and_usage.rst

```diff
@@ -0,0 +1,93 @@
+Configuration and usage
+=======================
+
+Getting around administrator interface
+--------------------------------------
+
+`Edx-enterprise` is integrated with edx-platform Django admin. It can be accessed at
```

Comment:
How about: "`edx-enterprise` is integated with edx-platform's django admin site. It can be accessed at `$LMS_SERVER_ADDRESS/admin/enterprise` ~~URL~~"